ec2api/api/network_interface.py

```diff
@@ -142,6 +163,21 @@ def create_network_interface(context, subnet_id,
         network_interface_id = network_interface['id']
         neutron.update_port(os_port['id'],
                             {'port': {'name': network_interface_id}})
+
+
+        ## Adding PNI entry if account is paas
+        if (original_context.paas_account and # (ec2utils.is_paas(original_context) and
+               original_context.project_id != context.project_id) :
+
+            pni = db_api.add_item(original_context,'pni',
+                              { 'os_data' : os_port['id'] })
+            cleaner.addCleanup(db_api.delete_item,
+                           original_context, pni['id'])
+            network_interface["pni"] = pni["id"]
+            db_api.update_item(context, network_interface)
```

Comment:
we can avoid this additional db update if we can include 'pni' in network_interface = db_api.add_item
https://github.com/hmdesai89/ec2-api/blob/393d200071a0c73bdba3bb02ae3f4fbd657368df/ec2api/api/network_interface.py#L154